hw/top_englishbreakfast/util/prepare_sw.py

```diff
@@ -29,9 +29,9 @@
 ]
 
 BAZEL_BINARIES = [
-    '//sw/device/lib/testing/test_rom',
+    '//sw/device/lib/testing/test_rom:test_rom',
```

Comment:
Don't these mean the same thing?

Reply:
Yes - I've changed it back to the more concise form